src/options.js

```diff
@@ -28,6 +28,7 @@ import language from './language';
  * @property {boolean} [optOut=`false`] -  Whether or not to disable tracking for the current user.
  * @property {function} [onError=`() => {}`] - Function to call on error.
  * @property {function} [onExitPage=`() => {}`] - Function called when the user exits the browser. Useful logging on page exit.
+ * @property {Object} [plan = {branch: '', source: '', version: ''}] - Observe plan information.
```

Comment:
Might be good to call out the expected values. For JSDoc we could use this notation so its not all squashed on one line
```
   * @property {Object} [plan] Tracking plan properties
   * @property {string} [plan.branch] The tracking plan branch name e.g. "main"
   * @property {string} [plan.source] The tracking plan source e.g. "web"
   * @property {string} [plan.version] The tracking plan version e.g. "1", "15"
```

Reply:
That sounds good. I can adjust the comment for this. One minor thing for the word `tracking plan`, just in current SDK we have a `tracking option`, but these two are not actually related.